src/state/actions/passwordReset.ts

```diff
@@ -0,0 +1,66 @@
+import railsAxios from '@util/railsAxios';
+
+interface ResetPasswordPropsBase {
+	onComplete: () => void;
+	input: string;
+	setIsSubmitting: (value: boolean) => void;
+	setError: (value: string) => void;
+}
+
+export type RequestResetTokenProps = ResetPasswordPropsBase;
+
+export const requestResetToken = async (store, {
+	onComplete, input, setIsSubmitting, setError,
+}: RequestResetTokenProps) => {
+	const { userIdentity } = store.state;
+	const endpoint = `http://localhost:3000/password_resets/${userIdentity}/`;
```

Comment:
You can use the  `API_BASE_URL` env variable we have to handle different environments (loaded from `environment.ts`).

Reply:
Fixed! Thanks :)